translations/zh_Hant.po

```diff
@@ -0,0 +1,3854 @@
+msgid "Public event"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "list this event on this site and search engines"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Private event"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "do not list this event publicly"
+msgstr "不要公開列出此活動行程"
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Select Event Type"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Event Topic"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Select Event Topic"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Event Sub-topic"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Select Event Sub-Topic"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Add Code of Conduct"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Code of Conduct"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "License of Event Data & Content"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Licenses"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Select License"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Forward"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Save draft"
+msgstr ""
+
+#: app/templates/components/forms/wizard/basic-details-step.l10n.hbs:1
+msgid "Publish"
+msgstr "發布"
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "turn off"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "turn on"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Session, Microlocations, Session Types"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Preview Text"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Session Types"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Call for speakers"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Announcement"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Private Link"
+msgstr ""
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "Copy"
+msgstr "複製"
+
+#: app/templates/components/forms/wizard/sessions-speakers-step.l10n.hbs:1
+msgid "S
```

Comment:
@niranjan94 This is the first time I am looking at translation files so a bit confused, why are some of the `msgstr` randomly empty